src/metrics.js

```diff
@@ -0,0 +1,81 @@
+const octokit = require('@octokit/rest')() 
+const fs = require('fs');
+ 
+var gitToken = "thisToken"
+ 
+octokit.repos.getForOrg({
+  org: 'octokit',
+  type: 'public'
+})
+ 
+var targetProjectRoot = "C:/Users/SHAWN/data/"
+var github = { "owner": "deepanjali19", "repo": "GitHub-Dashboard", "branch": "master" }
+ 
+downloadGitHubRepo(github, targetProjectRoot)
+ 
+async function downloadGitHubRepo(github, targetDirectory) {
```

Comment:
An `async` function needs to return a `Promise`.  You should add a `return` on line 22 to achieve this.